src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.html

```diff
@@ -40,8 +40,7 @@ <h5 class="comment-authr-name">
                 </h5>
 
                 <small class="createTimeBlock" id="{{'comment_time_' + counter}}">
```

Comment:
this should just have the timestamp, aligned to the right of the comment block (taking into account the kebab menu, when applicable).